server.js

```diff
@@ -7,7 +7,9 @@ new WebpackDevServer(webpack(config), {
   contentBase: baseConfig.output.path,
   publicPath: baseConfig.output.publicPath,
   hot: process.env.NODE_ENV === 'development' ? true : false,
-  historyApiFallback: true,
+  historyApiFallback: {
+    disableDotRule: true,
+  },
```

Comment:
I actually don't know why this is here. We don't need any fallback, as far as I can tell.